### DISCUSSION TITLE
Is it abuse behavior to use Github issues and discussions as blog?

### DISCUSSION THREAD

**Discussion by Xuanwo** [community/community#23508] [New to GitHub]
<p>Hi,</p>
<p>I do know we have Github pages here. But I’m interested is it abuse behavior to use Github issues and discussions as blogs? I can’t find a clear statement about this from Github’s ToS.</p>
<p>Thanks in advance.</p>

**Comment by jsoref** [comment]
<p>If they’re relevant to your repository, …<br>
But what happens if someone else posts an issue or discussion item, are you suddenly “blogging” their “content”? Isn’t that a risky proposition?</p>
<p>That said, have you looked at:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://dev.to/">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/2119212/181077464-09bac61b-f209-4381-8e54-cfe949c04b1a.png" class="site-icon" width="32" height="32">

      <a href="https://dev.to/" target="_blank" rel="noopener">DEV Community</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/2119212/181077465-3ccf3cd5-c25b-44ec-8fdc-47ab63cf6d00.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/f/1f865d6c4c636551ac83a1cb2e78d76a3f4bde9b_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/f/1f865d6c4c636551ac83a1cb2e78d76a3f4bde9b_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/f/1f865d6c4c636551ac83a1cb2e78d76a3f4bde9b.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/f/1f865d6c4c636551ac83a1cb2e78d76a3f4bde9b_2_10x10.png"></div>

<h3><a href="https://dev.to/" target="_blank" rel="noopener">DEV Community</a></h3>

  <p>A constructive and inclusive social network for software developers. With you every step of your journey.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by jfullstackdev** [comment, marked as answer]
<p>nope, actually, they encourage to do that, as long as the content is not abusive or in conflict with GitHub ToS,</p>
<p>too many users use GitHub because of GitHub Pages and the Discussion Tab,</p>
